### DISCUSSION TITLE
CNAME 记录为 `cc-ecdn.net` / `cc-ecdn.net.cdn.cloudflare.net` 的域名似乎无法被替换 IP 地址

### DISCUSSION THREAD

**Discussion by cherinyy** [XIU2/CloudflareSpeedTest#473] [分享]
示例：`store.ubisoft.com`

```
> nslookup store.ubisoft.com one.one.one.one
服务器:  one.one.one.one
Address:  2606:4700:4700::1111

非权威应答:
名称:    commcloud.prod-abbs-ubisoft-com.cc-ecdn.net.cdn.cloudflare.net
Addresses:  104.18.64.55
          104.18.65.55
Aliases:  store.ubisoft.com
          commcloud.prod-abbs-ubisoft-com.cc-ecdn.net
```

在替换 IP 地址后，网站将无法被打开，报错“Error 1001 - DNS resolution error”。

![image](https://github.com/XIU2/CloudflareSpeedTest/assets/9050713/8e821053-229e-46e9-ab11-bb697d6efa13)

这似乎是因为 CNAME 记录为 `cc-ecdn.net` / `cc-ecdn.net.cdn.cloudflare.net` 的服务实际由 Salesforce Commerce Cloud [提供](https://developers.cloudflare.com/cloudflare-for-platforms/cloudflare-for-saas/saas-customers/provider-guides/salesforce-commerce-cloud/)。

不知道是否还有其他类似的情况？

**Comment by XIU2** [comment]
个别使用 Cloudflare CDN 付费套餐的域名只能通过其默认解析 IP 访问。

这种我倒是见过几次，常出现在一些游戏服务上，像你这个属于游戏网站，还有一些是游戏里面连接的服务域名也会这样。
